tests/http/auth-jwt-file.yml

```diff
@@ -0,0 +1,69 @@
+config: !file tests/http/config/auth-jwt-file.graphql
+name: Auth with JWT loaded from file
+assert:
+  - request:
+      method: POST
+      url: http://localhost:8080/graphql
+      body:
+        query: |
+          query {
+            public_scalar
+            private_nested_field {
+              name
+            }
+          }
+    response:
+      body:
+        data:
+          public_scalar: data from public scalar
+          private_nested_field:
+            name: private nested name
+
+  - request:
+      method: POST
+      url: http://localhost:8080/graphql
+      body:
+        query: |
+          query {
+            private_scalar
+          }
+    response:
+      body:
+        data: null
+        errors:
+          - message: |
+              Validation Error
+              • Haven't found auth parameters []
```

Comment:
Create a GraphQL response error type to send such errors. Take inspiration from other projects. 
The main idea is that the errors should be structured and easy to parse for the consume of the API.

Reply:
I've tried to use `async_graphql::Error` to make it more graphql way, but due to having auth validation logic wrapped in expr that operates on arbitrary anyhow::Error that will require some refactor of this code to be able to use it. I've simplified the error here and we can think about error improvements overall across the project later